scripts/release/README.md

```diff
@@ -107,13 +113,13 @@ Once you have cherry picked all of the commits you want to include in the releas
 git push origin 16.8.3
 ```
 
-Once CI is complete, follow the regular [**next**](#publishing-next) and [**promote to stable**](#publishing-a-stable-release) processes.
+Once CI is complete, follow the regular [**next**](#publishing-release) and [**promote to stable**](#publishing-a-stable-release) processes.
 
 <sup>1: The `build-info.json` artifact can also be used to identify the appropriate commit (e.g. [unpkg.com/react@16.8.3/build-info.json](https://unpkg.com/react@16.8.3/build-info.json) shows us that react version 16.8.3 was created from commit [`29b7b775f`](https://github.com/facebook/react/commit/29b7b775f)).</sup>
 
 # Scripts
 
-## `create-next`
+## `build-release`
```

Comment:
I forgot this script existed 😮Can we call it something less ambiguous, like `build-release-locally` or something? Just so it doesn't get confused with `prepare-release`.

I don't think I've ever used it because pushing to a PR to kick off a CI build tends to be faster even with all the extra checks because of the build parallelization.

Reply:
Sure. Renamed